shared/middleware/helpers/auth_helpers.rb

```diff
@@ -0,0 +1,31 @@
+#
+# Utility methods that help middleware access dashboard authentication and
+# permissions information.
+#
+# Note: This file should be loaded in the context of a Sinatra application;
```

Comment:
how about "Sinatra application, such as Pegasus", for those only kind of familiar with Pegasus.